package.json

```diff
@@ -32,11 +32,18 @@
     "axios": "^0.19.2",
     "babel-loader": "^8.0.0",
     "classnames": "^2.2.6",
+    "concurrently": "^4.1.2",
```

Comment:
I think you meant to add the packages in the `server` package.json. This is the react project